modules/agentless-scanner-role/main.tf

```diff
@@ -51,6 +51,18 @@ data "aws_iam_policy_document" "scanner_policy_document" {
     effect    = "Allow"
     actions   = ["sts:AssumeRole"]
     resources = var.account_roles
+
+    condition {
+      test     = "StringEquals"
+      variable = "iam:ResourceTag/DatadogAgentlessScanner"
+      values   = ["true"]
+    }
+
+    condition {
```

Comment:
should this be a dynamic block? if `var.organization_unit_ids` is empty, it looks like this condition would never pass?

Reply:
oh, I see the wildcard, but perhaps better not having the condition at all in that case then (?)

Reply:
The wildcard is the default condition. That what appeared to be the most simple to me. I'm not sure how to do that using condition.